src/core/AdminGuesser.tsx

```diff
@@ -98,7 +105,6 @@ const AdminGuesser = ({
       queryClient={queryClient}
       theme={theme}
       darkTheme={darkTheme}
-      lightTheme={lightTheme}
```

Comment:
`lightTheme` isn't documented in react-admin - `theme` is the prop to use. 

https://marmelab.com/react-admin/Admin.html#theme